test/cqlpy/test_materialized_view.py

```diff
@@ -1538,3 +1538,12 @@ def test_alter_table_add_select_star(cql, test_keyspace):
             cql.execute(f'INSERT INTO {base} (p,a,b,c) VALUES (0,1,2,3)')
             assert {(0,1,2,3),(1,2,3,None)} == set(cql.execute(f"SELECT p,a,b,c FROM {base}"))
             assert {(0,1,2,3),(1,2,3,None)} == set(cql.execute(f"SELECT p,a,b,c FROM {mv}"))
+
+# Test view representation in system.* tables
+def test_view_in_system_tables(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int") as base:
+        with new_materialized_view(cql, base, '*', 'v,p', 'v is not null and p is not null') as view:
+            res = [ f'{r.keyspace_name}.{r.view_name}' for r in cql.execute('select * from system.built_views')]
+            assert view in res
+            res = [ f'{r.table_name}.{r.index_name}' for r in cql.execute('select * from system."IndexInfo"')]
```

Comment:
As I noted elsewhere, I don't understand the point of this check - you have no reason to worry that a random view will be listed in IndexInfo, and more importantly - have no idea what name to even look for in IndexInfo.
But I guess an extra assert does no real harm, even if I don't think it has any chance of catching a bug.